addons/addons.json

```diff
@@ -6,6 +6,7 @@
   "scratch-notifier",
   "discuss-button",
   "editor-stepping",
+  "invert-stage",
```

Comment:
This should be added before the comment that says `"// NEW ADDONS ABOVE THIS ↑↑"`, not in the middle of the addon list..